docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -159,15 +159,17 @@ Overview of :code:`vivarium_inputs`
 
 Some important notes and considerations not included in the documentation above are listed below:
 
-.. todo::
-   
-   List default behavior of get_measures/other functions once the GBD 2021 update is finalized, including things like:
+   - All data returned is filtered to 500 draws (draw 0 through 499), even if more draws are available
+   - Returns data for all most-detailed age groups and sexes - if any such data is missing in GBD, NaNs will be filled with zeros
+   - Returns *default* version IDs within GBD system
+   - Returns data specific to most recent *published* year unless user specifies to return all available years 
+
+      - Note: will return data specific to 2021 for GBD 2021, despite estimates being available for 2022 because the 2022 year was not published as part of GBD 2021)
```

Comment:
```suggestion
      - Note: will return data specific to 2021 for GBD 2021, despite estimates being available for 2022 because the 2022 year was not published as part of GBD 2021
```